pkg/controller/nodes/common/utils.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"github.com/lyft/flytepropeller/pkg/apis/flyteworkflow/v1alpha1"
+	"github.com/lyft/flytepropeller/pkg/controller/executors"
+	"github.com/lyft/flytepropeller/pkg/utils"
+	"strconv"
+)
+
+func GenerateUniqueId(parentInfo executors.ImmutableParentInfo, nodeID string) (string, error) {
```

Comment:
lets add a comment explaining what this is

Reply:
YES